.changeset/purple-jars-pay.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': patch
```

Comment:
Does astro expose some `vite` APIs? If so, this should count as `minor`.

Reply:
I guess having the vite config inside astro config expose vite apis https://docs.astro.build/en/guides/configuring-astro/#customising-output-filenames

Reply:
Yeah I think it'll only be through the `vite` config option, but since users could already use Vite 4.3 by running `npm update vite` to change the version Astro will use, I made it as a patch.